### DISCUSSION TITLE
Speed of response on issues is blazingly slow 😞

### DISCUSSION THREAD

**Discussion by hrynevychroman** [parcel-bundler/parcel#9115] [Q&A]
Don't want to spam, but my issue is opened for second week and don't reviewed by anyone. Some issues that were post after my have response from dev team, maybe i don't know something and someone can help me. Without that damn sourcemap for styles i can't start my project. I am very excited with power of parcel, Rust is blazingly fast, but feedback of dev team is blazingly slow. Don't sure that in critical situations someone can help us in time.

Issue link: https://github.com/parcel-bundler/parcel/issues/9099

**Comment by devongovett** [comment]
Apologies, we're all very busy. No one is working on the project full time, we all have jobs and other responsibilities. For me it's basically nights and weekends when I have time. Therefore we're unable to respond to every issue immediately, it'll take some time for someone to have bandwidth to look at it. Sorry about that, it's just the nature of free open source software I guess.

**Reply by hrynevychroman** [reply]
Hello @devongovett, i'm very glad to hear you. Yeah, i understand you that this is mostly the issue with open source software. I want to say thank you for all that titanic work with Parcel (and other great projects you are working on) and will wait for your response on that issue. Have a nice day ❤️

**Comment by hrynevychroman** [comment]
@devongovett you asked me for link to codesanbox, i give you it asap and nothing new during 4 days, if it possible, can you give ETA and Deadline for your review?
